python/python/ert/test/extended_testcase.py

```diff
@@ -56,10 +56,10 @@ def __init__(self , *args , **kwargs):
         self.__share_root = None
         installAbortSignals()
         super(ExtendedTestCase , self).__init__(*args , **kwargs)
-        self.__str__ = self.__repr__
 
-    def __repr__(self):
-        return 'ExtendedTestCase( TESTADATA_ROOT=%s, SOURCE_ROOT=%s, SHARE_ROOT=%s, BUILD_ROOT=%s)' % (TESDATA_ROOT,
+
+    def __str__(self):
```

Comment:
This should be `__repr__`.  `str(obj)` will fallback to `__repr__` if `__str__` is not defined, not the other way.